MekHQ/src/mekhq/gui/RepairTab.java

```diff
@@ -766,21 +772,22 @@ public void focusOnUnit(UUID id) {
         if (row != -1) {
             servicedUnitTable.setRowSelectionInterval(row, row);
             servicedUnitTable.scrollRectToVisible(servicedUnitTable.getCellRect(row, 0, true));
+            return true;
         }
+        return false;
     }
 
     public void refreshServicedUnitList() {
-        int selected = servicedUnitTable.getSelectedRow();
+        UUID uuid = null;
+        if (null != getSelectedServicedUnit()) {
+            uuid = getSelectedServicedUnit().getId();
+        }
```

Comment:
```suggestion
        UUID uuid = (getSelectedServicedUnit() != null) ? getSelectedServicedUnit().getId() : null;
```

Reply:
Sure, was simply keeping the source in line with existing code.